seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/SinkAggregatedCommitterTask.java

```diff
@@ -118,16 +115,7 @@ public void init() throws Exception {
         this.commitInfoSerializer = sink.getSink().getCommitInfoSerializer().get();
         this.aggregatedCommitInfoSerializer =
                 sink.getSink().getAggregatedCommitInfoSerializer().get();
-        if (this.aggregatedCommitter instanceof SupportResourceShare) {
-            resourceManager =
-                    ((SupportResourceShare) this.aggregatedCommitter)
-                            .initMultiTableResourceManager(1, 1);
-        }
         aggregatedCommitter.init();
-        if (resourceManager != null) {
-            ((SupportResourceShare) this.aggregatedCommitter)
```

Comment:
This can trigger the method org.apache.seatunnel.connectors.seatunnel.jdbc.sink.JdbcSinkWriter#setMultiTableResourceManager which is in jdbcsink that can create a SimpleJdbcConnectionPoolProviderProxy.